pulse_actions/authentication.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+class AuthenticationError(Exception):
```

Comment:
Thinking more about it, CredentialsError also has the same problem, and I can't think of anything that is not misleading. Let's just leave it as AuthenticationError.